cirq-google/cirq_google/ops/common_gate_families_test.py

```diff
@@ -0,0 +1,259 @@
+# Copyright 2021 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for Common Gate Families used in cirq-google"""
+
+import pytest
+import sympy
+import numpy as np
+import cirq
+import cirq_google
+
+ALL_POSSIBLE_FSIM_GATESS = [
+    cirq.CZPowGate,
+    cirq.FSimGate,
+    cirq.PhasedFSimGate,
+    cirq.ISwapPowGate,
+    cirq.PhasedISwapPowGate,
+    cirq.IdentityGate,
+]
+THETA, PHI = sympy.Symbol("theta"), sympy.Symbol("phi")
+VALID_IDENTITY = [
+    (cirq.CZ ** THETA, {THETA: 2}),
+    (cirq.IdentityGate(2), {}),
+    (cirq.FSimGate(THETA, PHI), {THETA: 0, PHI: 0}),
+    (cirq.PhasedFSimGate(THETA, 0, 0, 0, PHI), {THETA: 0, PHI: 0}),
+    (cirq.ISWAP ** THETA, {THETA: 4}),
+    (cirq.PhasedISwapPowGate(exponent=THETA, phase_exponent=PHI), {THETA: 4, PHI: 2}),
+]
+
+VALID_CZPOW_GATES = [
+    (cirq.CZPowGate(exponent=THETA, global_shift=2.5), {THETA: 0.1}),
+    (cirq.CZ ** THETA, {THETA: 0.5}),
+    (cirq.FSimGate(theta=THETA, phi=0.1), {THETA: 0}),
+    (cirq.FSimGate(theta=2 * np.pi, phi=PHI), {PHI: -0.4}),
+    (
+        cirq.PhasedFSimGate.from_fsim_rz(
+            theta=THETA,
+            phi=10,
+            rz_angles_before=(THETA, THETA),
+            rz_angles_after=(PHI, PHI),
+        ),
+        {THETA: 0, PHI: 2 * np.pi},
+    ),
+    (
+        cirq.PhasedFSimGate.from_fsim_rz(
+            theta=THETA,
+            phi=PHI,
+            rz_angles_before=(2 * np.pi, 2 * np.pi),
+            rz_angles_after=(0, 0),
+        ),
+        {THETA: 2 * np.pi, PHI: -0.01},
+    ),
+] + VALID_IDENTITY
```

Comment:
Can use literal unpacking here:
```suggestion
    *VALID_IDENTITY,
]
```
Also in a few places below.